postgresql/db-index-status.yml

```diff
@@ -0,0 +1,57 @@
+type: db
+data:
+  query: SELECT * FROM pg_stat_all_indexes;
+  dbUrl: jdbc:postgresql://${POSTGRESQL_DB_HOST_PORT}
+  dbDriverClass: org.postgresql.Driver
+  dbUser: ${POSTGRESQL_DB_USER}
+  dbPassword: ${POSTGRESQL_DB_PASSWORD}
+  dbAdditionalConnectionParams: ${POSTGRESQL_DB_ADDITIONAL_PARAMS}
+
+observation:
+  - name: indexes
+    metricNamespace: postgresql
+
+    metric:
+      - name: indexes.all.table
+        source: relname
+        type: text
+        send: false
+        label: Table name
+
+      - name: indexes.all.schema
+        source: schemaname
+        type: text
+        send: false
+        label: Schema name
+
+      - name: indexes.all.index
+        source: indexrelname
+        type: text
+        send: false
+        label: Index name
+
+      - name: indexes.all.scan
+        source: idx_scan
+        type: long_counter
+        label: Index scan
+        description: Number of index scans initiated on this index
+
+      - name: indexes.all.rows.read
+        source: idx_tup_read
+        type: long_counter
+        label: Returned index entries
+        description: Number of index entries returned by scans on this index
+
+      - name: indexes.all.rows.feched
+        source: idx_tup_fetch
+        type: long_counter
+        label: Fetched rows
+        description: Number of live rows fetched by index scans
+
+    tag:
+      - name: indexes.all.table
```

Comment:
I guess this is just a plain "table" tag, not some special "index all table" that has no relation to DB tables. If yes, name should be `postgresql.table`.